lib/workload/stateless/stacks/metadata-manager/app/models/lab/library.py

```diff
@@ -25,7 +25,7 @@ class Phenotype(models.TextChoices):
 
 
 class Quality(models.TextChoices):
-    VERY_POOR = 'very-poor'
+    VERY_POOR = 'very-poor', 'VeryPoor'
```

Comment:
We should fix those inconsistencies in the source..... 
I'll make a note... somewhere...